src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -3744,7 +3745,14 @@ private void BuildInlineLocalizedProperties(IIndentedStringBuilder writer, XamlO
 					var localizedValue = BuildLocalizedResourceValue(null, prop, objectUid);
 					if (localizedValue != null)
 					{
-						writer.AppendLineInvariantIndented("{0} = {1},", prop, localizedValue);
+						if (isInInitializer)
+						{
+							writer.AppendLineInvariantIndented("{0} = {1},", prop, localizedValue);
+						}
+						else
+						{
+							writer.AppendLineInvariantIndented("{0} = {1};", prop, localizedValue);
```

Comment:
This is the same line as the "if" condition. (fyi ReSharper would warn about this)